changelog/2.006.dd

```diff
@@ -31,8 +31,7 @@ $(WHATSNEW
 	$(LI Added the
 	$(LINK2 $(ROOT_DIR)phobos/std_variant.html, std.variant) module.)
 	$(LI Incorporated many of the Tango GC structural differences (much more to go still).)
-	$(LI Added the
-	$(LINK2 $(ROOT_DIR)phobos/std_contracts.html, std.contracts) module.)
+	$(LI Added the $(D std.contracts) module.)
```

Comment:
> I guess we could link to Seb's new doc archive, but I'm not sure how stable those links would be.

If we can upstream Seb's doc archive, then this certainly sounds like a very good idea.